articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
-* **Allowed Callback URLs**: the URLs of your application(s) to which Auth0 can redirect the user after authentication.
-* **Allowed Logout URLs**: the URLs that Auth0 can redirect your users to after logging out;
-* **Allowed Origins (CORS)**: the URLs of the applications running your JavaScript code (prevents same-origin policy errors when using Auth0 from within a web browser)
-* **JWT Expiration (Seconds)**: the amount of time before the Auth0 access tokens expires
-* **Use Auth0 instead of the IdP to do Single Sign On**: if enabled, this setting prevents Auth0 from redirecting authenticated users with valid sessions to the identity provider (such as Facebook, ADFS, and so on)
+The client type should match your application:
 
-## Addons
+- **Native**: Used for mobile, desktop or hybrid apps, than run natively in a device, like Android, Ionic or iOS. For a complete listing of the SDKs Auth0 offers for mobile apps refer to: [Native SDKs](/quickstart/native).
 
-![](/media/articles/applications/applications-addon-types.png)
+- **Single Page Web Applications**: Used for JavaScript front-end apps that run on a browser, like Angular, jQuery or React. For a complete listing of the SDKs Auth0 offers for SPAs refer to: [Single Page App SDKs](/quickstart/spa).
 
-Addons are extensions associated with clients. They are typically third-party APIs used by the application(s) for which Auth0 generates access tokens. Some typical scenarios used include:
+- **Regular Web Applications**: Used for traditional web applications that run on a server, like ASP .NET, Java or Node.js. For a complete listing of the SDKs Auth0 offers for Web Apps refer to: [Web App SDKs](/quickstart/webapp).
 
-* **Accessing External APIs**: Using the Delegation endpoint, you can exchange a Client's access token for a third-party service's (such as Salesforce or Amazon) access token
-* **Integrating with Applications Using SAML2/WS-Federation**: Addons allow you to integrate with any custom or SSO integration that does not currently enjoy built-in Auth0 support, since they allow you to configure every aspect of the SAML2/WS-Federation integration.
+- **Non Interactive Clients**: Used for server to server applications like CLIs, daemons or services running on your backend. Typically you would use this option if you have a service that requires access to an API and you want it to request access tokens in order to do so.
 
-![](/media/articles/applications/applications-sso-integrations-overview.png)
+After you set the name and client type, click **Create**.
 
-## Connections
+Each client has four available tabs:
 
-![](/media/articles/applications/applications-connections-example.png)
+- [Quick Start](${manage_url}/#/clients/${account.clientId}/quickstart): Lists all available Quick Starts, filtered by your client's type.
```

Comment:
I'm getting this

![image](https://cloud.githubusercontent.com/assets/1980994/19745246/41eb5c14-9ba7-11e6-86eb-2ec554f64808.png)